### ISSUE TITLE
RV 508 Phase 2 

### ISSUE THREAD

**Issue by cthomasONRR** [ONRR/onrr.gov-site#1024]
RV is sending in their last Phase 2 documents for the 508 Compliance project.

**Comment by cthomasONRR** [comment]
Jodie submitted the following 2 documents:

- [ ] [Indian-Payor-Handbook-Chapter-1-About-this-Handbook-9.2018.pdf](https://github.com/ONRR/onrr.gov-site/files/7180770/Indian-Payor-Handbook-Chapter-1-About-this-Handbook-9.2018.pdf)
- [ ] [Processed Gas Example Final 05262020.pdf](https://github.com/ONRR/onrr.gov-site/files/7180771/Processed.Gas.Example.Final.05262020.pdf)

I have the Indian Payor Handbook updated, but the Processed Gas Example has 4 character encoding errors. I'm currently trying to figure out if we can mark this document as compliant or not.

**Comment by cthomasONRR** [comment]
I sent the following email to Sid Sharma and Regina Wendling:

"Hi Sid and Regina,

We have one document that Jodie Peterson has worked extremely hard at making compliant. There is one remaining issue within the document that we cannot resolve though:

There are 4 character encoding errors in math equations due to the presence of parentheses that Adobe doesn’t like. Jodie put in alt text that those characters will not be read by the screen reader. Is that an acceptable solution, and can we mark this document as being compliant? 

We have all tried different methods of resolving this parentheses issue, and none of us have found a way to resolve this ‘fail’.

Could you please review the attached document, and let us know if we can classify it as compliant?

Thanks for your time and help,

Christine Thomas"

**Comment by cthomasONRR** [comment]
Regina sent this email as a response:

"Hi, Christine:

The character encoding error usually is a font issue.  Try extracting the page, saving it as a Word file, then running the Inspect document and change the font for the items, reconvert to PDF, then run accessibility checker to see if issue is resolved.  Keep in mind that new errors may arise after your insert the new page back into the main PDF, but should be easily corrected.

I also noticed your footnotes are not tagged; therefore, are not being read.  Should they be read/tagged? 

Sincerely,

Regina A. Wendling"

**Comment by cthomasONRR** [comment]
I replied back to Regina and Sid:

"Hi Regina and Sid,

Since this issue is occurring within a math equation, there is no font option to change. Jodie has worked with a specialist at Microassist about this specific issue, and they did not have a solution for us. Our current workaround is having alt text and a footnote for this equation… since the footnote and the alt text give the same information as the math equation, is this an acceptable solution and can we mark this document as being compliant?

If you do not think this is an acceptable solution, could you please take a look and see if you can resolve it? We feel that we have exhausted all our resources in trying to resolve this ourselves. There will be more documents like this in the future, so we are trying to find a real solution here.

Thanks so much for your time and help,

Christine Thomas"

**Comment by cthomasONRR** [comment]
Jodie emailed the following document. It passed 508 accessibility check. I updated the file on the dev and live sites, and updated the accessibility tracking spreadsheet:

- [ ] [Indian-Payor-Handbook-Chapter-3-Oil-Valuation-2.2019](https://app.zenhub.com/files/210892300/926c88a7-9f99-4da9-9f5e-41bb2029561c/download)

**Comment by cthomasONRR** [comment]
Regina was able to resolve the remaining issues within the processed gas example. I just updated it on the dev and live sites. Closing this issue.

- [ ] [Processed Gas Example Final 05262020](https://app.zenhub.com/files/210892300/2ca4f5c5-bb91-4de5-948e-f681b1374444/download)
